tests/project/project_1/project.yml

```diff
@@ -0,0 +1,22 @@
+native_app:
+  name: myapp
+
+  # the following shows how to properly use a quoted identifier in YAML
+  source_stage: >-
+    "MySourceSchema"."SRC_Stage"
```

Comment:
Why we limit our selves to only schema.object identifier instead of db.schema.object?

Reply:
We are uploading all artifacts to the schema within the application package db. We already know the name of the app package that will be, hence schema and stage are the unknowns.